script.js

```diff
@@ -0,0 +1,8 @@
+(function getNavbar(){
+    var theScriptHTML = document.getElementById('navbar-template').innerHTML;
```

Comment:
You could have a look at keeping the template in an external file and loading it in rather than using a script tag on the page. I can't remember how to do this. I will have a go and throw something up.